### DISCUSSION TITLE
🚀 What’s new on daily.dev - Product Updates v.2.27.0

### DISCUSSION THREAD

**Discussion by nimrodkra** [dailydotdev/daily#227] [General]
# We have some cool things to share with you
Some of you may have already noticed that we released new stuff recently, but anyway let’s recap on what’s new! We built some cool features that aim to make daily.dev more fun to use!

Major features are:
* Real-time trends 🔥
* Real-time interactions 🔆
* Reading ranks 💎

Read all about it and check out how it looks like in our [official product update post](https://daily.dev/posts/whats-new-on-daily-dev-product-updates-v-2-27-0).

**Comment by Kvaibhav01** [comment]
I really liked the idea of adding ranks. Because those who read articles frequently can now be more encouraged to read even more articles! The more info you get the better it is! 🎸

And those who were not reading for any reason before can now learn more. 🔥

**Reply by otisgbangba** [reply]
I absolutely agree with you on that. Thanks for sharing.

**Comment by ryota-murakami** [comment]
Neat! 
I'm enjoy your newsletter in these days, thanks!

**Comment by oscarrodar** [comment]
This is a great idea! Thanks!

**Comment by mheob** [comment]
You are on a great way. I enjoy daily.dev every morning like I used to read my newspaper.
I thank you guys for that. Keep up the good work!
